components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -108,23 +110,36 @@
 Port = Annotated[int, Field(gt=0, lt=2**16)]
 ZstdCompressionLevel = Annotated[int, Field(ge=1, le=19)]
 
+StrEnumSerializer = PlainSerializer(serialize_enum)
+
+PathStr = Annotated[pathlib.Path, PlainSerializer(serialize_path)]
```

Comment:
the annotated `pathlib.Path` is not a `str` so it might be misleading to name it `PathStr`. Do you think it would be better to name it as `SerializablePath`?